.github/workflows/deploy.yml

```diff
@@ -14,6 +14,13 @@ on:
       title:
         description: 'Pull Request Title'
         required: false
+      release_tag:
+        description: 'Name of Release Tag'
+        required: false
+      stop:
```

Comment:
this should probably be `stop_after_release` just for future clarity

Reply:
Agreed